subsys/bluetooth/host/iso.c

```diff
@@ -2024,7 +2066,7 @@ static bool valid_cig_param(const struct bt_iso_cig_param *param, bool advanced,
 #endif /* CONFIG_BT_ISO_TEST_PARAMS */
 
 	return true;
-}
+	}
```

Comment:
Patch needs a clang-format-diff pass :)

Reply:
The formatter in VSCode is having a hard time with files with >1000 lines :(